make/photon/common/install_cert.sh

```diff
@@ -2,14 +2,14 @@
 
 set -e
 
-if [ ! -f /etc/pki/tls/certs/ca-bundle.crt.original ]; then
-    cp /etc/pki/tls/certs/ca-bundle.crt /etc/pki/tls/certs/ca-bundle.crt.original
+if [ ! -f ~/ca-bundle.crt.original ]; then
```

Comment:
Is this supposed to be "-/ca-bundle.crt" looks like a typo with the leading -

Reply:
No, ~ means user's home directory

Reply:
Nvm, wasn't wearing my glasses 😂